app/javascript/lib/cell-faceting.js

```diff
@@ -150,7 +135,8 @@ export function filterCells(
   const numFacetsSelected = Object.keys(selection).length
   const numFiltersSelected = Object.values(selection).length
   const filterLogProps = {
-    perfTime: filterPerfTime,
+    'perfTime': filterPerfTime,
+    'perfTime:counts': perfTimeCounts,
```

Comment:
Is this an "I'm curious how long it takes" or an "I'm concerned about how long it may take"?

Reply:
Mostly the former.  In my local testing, getting counts was typically < 50 ms.  Since I wrote the trivial timing code while benchmarking and it doesn't cost us anything to log the extra property, I thought I'd add it in for richer insight towards possible optimizations later.